nrf52840-hal/src/ieee802154.rs

```diff
@@ -26,9 +26,6 @@ pub struct Radio<'c> {
     _clocks: &'c (),
 }
 
-/// Default Clear Channel Assessment method = Carrier sense
-pub const DEFAULT_CCA: Cca = Cca::CarrierSense;
```

Comment:
let's leave this method as the default; let's undo this deletion

Reply:
it's re-added in line 42 (no idea why I moved the value down)